internal/kubernetes/operator/config_exporter.go

```diff
@@ -160,9 +160,20 @@ func (e *ConfigExporter) Run() (string, error) {
 }
 
 func (e *ConfigExporter) exportProject() ([]runtime.Object, string, error) {
+	atlasProject, err := e.dataProvider.Project(e.projectID)
+	if err != nil {
+		return nil, "", err
+	}
+	if e.orgID != "" && e.orgID != atlasProject.OrgId {
+		return nil, "", fmt.Errorf("%q is not the owner of project %s, org id %s is",
+			e.orgID, atlasProject.GetId(), atlasProject.OrgId)
+	}
+	e.orgID = atlasProject.OrgId
```

Comment:
If the check above is to ensure orgID is not empty and equal to the value in the project, why still assign it here?